src/main/kotlin/gdx/liftoff/actions/GlobalActionContainer.kt

```diff
@@ -67,8 +67,8 @@ class GlobalActionContainer : ActionContainer {
                 return false
             }
         }
-        if(!input.contains('.') || input.matches(Regex("(^\\.)|(\\.$)"))) {
-            return false;
+        if (!input.contains('.') || input.matches(Regex("(^\\.)|(\\.$)"))) {
+            return false
```

Comment:
Ha, you even cleaned up my lousy Java-style code in this Kotlin! Huzzah.

Reply:
![Don't thank me. Thank ktlintFormat.](https://user-images.githubusercontent.com/60154347/150317384-25206eb0-f496-494e-8b20-f5007647790e.png)